docs/src/modules/components/overview/Internationalization.tsx

```diff
@@ -290,7 +290,7 @@ function ValidationDemo() {
 
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <DemoWrapper link="/x/react-date-pickers/validation">
+      <DemoWrapper link="/x/react-date-pickers/validation/">
```

Comment:
This is the same case as the pricing page, but it's not as visible, Netlify has a bug: https://answers.netlify.com/t/no-trailing-slash-when-using-proxies/48790/11.